locales/cy.json

```diff
@@ -20,6 +20,12 @@
     "internal": "Mae problem gyda’r cyfleuster taliadau. Rhowch gynnig arall arni wedyn",
     "contactService": "Mae problem gyda’r cyfleuster taliadau. Rhowch gynnig arall arni wedyn"
   },
+  "paymentLinkError": {
+    "title": "Digwyddodd gwall:",
+    "invalidReference": "Rhif mae’n rhaid i fod yn 255 nod neu lai. Ni allwch ddefnyddio unrhyw rai o’r nodau canlynol < > ; : ` ( ) \" ' = &#124; \",\" ~ [ ] .",
+    "invalidAmount": "Cofnodwch swm mewn punnoedd a cheiniogau gan ddefnyddio digidau a phwynt degol, fel 123.45 neu 156.00 a yn £100,000 neu lai.",
+    "linkProblem": "Mae problem gyda'r ddolen sydd wedi cael ei hanfon atoch i'w defnyddio i dalu. Cysylltwch â'r gwasanaeth rydych yn ceisio gwneud taliad iddo."
```

Comment:
Can we use curly apostrophes please? `’` not `'`.

Reply:
Changed as suggested.